packages/protocol/migrations/08_reserve.ts

```diff
@@ -59,7 +60,9 @@ module.exports = deploymentForCoreContract<ReserveInstance>(
       await web3.eth.sendTransaction({
         from: network.from,
         to: reserve.address,
-        value: web3.utils.toWei(config.reserve.initialBalance.toString(), 'ether').toString(),
+        value: new BigNumber(
+          web3.utils.toWei(new BigNumber(config.reserve.initialBalance).toString(10), 'ether')
```

Comment:
Why change this?

Reply:
it doesn't work for large values because toString puts it in scientific notation.  since it appears this is only used for testing its probably unnecessary

Reply:
If it's a bug fix definitely keep it, thanks for the explanation